### PR TITLE
Added Left_View in Binany file

### PR DESCRIPTION
**Pull request by helios1101**
Added Left View in Binary File 
in ref to pr #509 
 - [x] Binary_Tree_LeftView

### CODE REVIEW

File: Binary_Tree_Top_View/Binary_Tree_Top_View.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+// function should print the topView of
+// the binary tree
+void topview(Node *root)
+{
+    if (root == NULL)
+        return;
+    queue<Node *> q;
+    map<int, int> m;
```
Comment (jainaman224) [inline Binary_Tree_Top_View/Binary_Tree_Top_View.cpp]: Use better variable name.

File: Binary_Tree_TopView/Binary_Tree_TopView.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+root.left.hd = hd – 1
+q.append(root.left)
+
+if(root.right):
+root.right.hd = hd + 1
+q.append(root.right)
+
+q.pop(0)
+for i in sorted (m):
+print(m[i], end = “”)
```
Comment (jainaman224) [inline Binary_Tree_TopView/Binary_Tree_TopView.py]: Indentation issues.

File: Binary_Tree_Top_View/Binary_Tree_Top_View.cpp (hunk 1 above)
Comment (helios1101) [reply Binary_Tree_Top_View/Binary_Tree_Top_View.cpp]: a queue is used as q in programming. So I changed the map to mp.

**Comment by helios1101** [comment]
Please add gssoc'19 label, as I'm a Gssoc aspirant.

File: Binary_Tree_Top_View/Binary_Tree_Top_View.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  q.pop(0)
+for i in sorted (m):
+  print(m[i], end = “”)
+
+# Driver Code
+if __name__ == ‘__main__’:
+
+
+root = newNode(1)
```
Comment (jainaman224) [inline Binary_Tree_Top_View/Binary_Tree_Top_View.py]: Indentation issue.

File: Binary_Tree_TopView/Binary_Tree_TopView.py (hunk 2 above)
Comment (helios1101) [reply Binary_Tree_TopView/Binary_Tree_TopView.py]: Done

File: Binary_Tree_Top_View/Binary_Tree_Top_View.py (hunk 3 above)
Comment (helios1101) [reply Binary_Tree_Top_View/Binary_Tree_Top_View.py]: Done

**Comment by jainaman224** [comment]
These are the files for top view which is already merged https://github.com/jainaman224/Algo_Ds_Notes/pull/629.

**Comment by helios1101** [comment]
This folder is being wrongly named as Top View, actually it's left View.

**Pull request merged**